src/Microsoft.Extensions.Primitives/project.json

```diff
@@ -1,11 +1,14 @@
 {
   "version": "1.0.0-*",
-  "description": "ASP.NET 5 primitives.",
+  "description": "ASP.NET Core primitives. Commony used types include:\r\nMicrosoft.Extensions.Primitives.IChangeToken\r\nMicrosoft.Extensions.Primitives.StringValues\r\nMicrosoft.Extensions.Primitives.StringSegment",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "https://github.com/aspnet/common"
-    }
+    },
+    "tags": [
+      "primitives"
```

Comment:
Oh, I see, this package is a Microsoft.Extensions package. I was confused because it's description specifically mentions ASP.NET Core. If these primitives are not ASP.NET Core specific, then should the description change?